department-of-veterans-affairs/caseflow
# Don’t show AMA decision Note section when empty

Resolves #8124 

### Description
If an AMA decision issue didn’t have any notes, the “Note”
section would be displayed as: `Note: “”`. If there aren’t any notes, we
don’t want to show the “Note” section.

### Testing Plan 
1. In `db/seeds.rb`, change line 348 such that notes are nil: `notes: nil` 
2. Run `RAILS_ENV=development bundle exec rake db:seed`
3. Run the app, and switch the user to an attorney with cases 
4. View your queue 
5. Click on `Bob Smith (975191063)`

Expected Result: You should not see a “Note:” section
